### PR TITLE
Expo: delete exporter functionality specific to native app export

### PR DESCRIPTION
**Pull request by bencodeorg**
Part two of deleting code related to our feature to export Applab/Gamelab to native apps (aka "Expo"). This PR removes the code paths of each app's Exporter that specifically supported exporting via Expo. 

## Links

- Part 1: https://github.com/code-dot-org/code-dot-org/pull/48186

## Testing story

Tested manually that the export features for Applab and Gamelab continued to function as they did before. I think they're both a bit broken at the moment (see [this Slack thread](https://codedotorg.slack.com/archives/C1B3PNDL7/p1663692678754429)), but the contents of the zip file that were produced by each was the same before and after this change (at least in terms of the list of files produced, I didn't verify that the contents of each file were identical). I did verify that the rendered page when opening index.html for each was the same, including for a project with its own assets.

## Follow-up work

Still a couple more things to do:
- ["generatedProperties" was added in the Expo work](https://github.com/code-dot-org/code-dot-org/pull/29029), but has its tendrils in other apps (eg, [poetry](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/studio/pages/init/loadPoetry.js#L15)). Want to remove that separately to make sure I don't cause any regressions.
- Delete expo wrapper code
- Remove other detritus

### CODE REVIEW

**Comment by bencodeorg** [comment]
Sorry, hold on review for a minute. Need to clean up some extraneous comments.

**Comment by bencodeorg** [comment]
Alrighty, should be all set!

File: apps/src/applab/applab.js [hunk 1]
```diff
@@ -392,6 +386,7 @@ Applab.init = function(config) {
   Applab.generatedProperties = {
     ...config.initialGeneratedProperties
   };
+  //remove
```
Comment (epeach) [inline apps/src/applab/applab.js:389]: Is this comment intended to leave? If we want to leave it here, we might want to add additional context about what is being removed, why, and when.

File: apps/src/p5lab/P5Lab.js [hunk 2]
```diff
@@ -571,54 +548,31 @@ export default class P5Lab {
...
-    project.projectChanged();
-    project.saveIfSourcesChanged();
-    getStore().dispatch(
-      setExportGeneratedProperties(this.generatedProperties.export)
+      getStore().getState().animationList
     );
   }
 
   /**
    * Export the project for web or use within Expo.
```
Comment (epeach) [inline apps/src/p5lab/P5Lab.js]: Can we update this comment to remove 'within Expo'?

File: apps/src/p5lab/gamelab/Exporter.js [hunk 3]
```diff
@@ -8,55 +8,37 @@ import * as assetPrefix from '@cdo/apps/assetManagement/assetPrefix';
 import download from '@cdo/apps/assetManagement/download';
 import exportGamelabCodeEjs from '@cdo/apps/templates/export/gamelabCode.js.ejs';
 import exportGamelabIndexEjs from '@cdo/apps/templates/export/gamelabIndex.html.ejs';
-import exportExpoPackageJson from '@cdo/apps/templates/export/expo/package.exported_json';
```
Comment (epeach) [inline apps/src/p5lab/gamelab/Exporter.js:11]: This clean-up looks so good!

File: apps/src/p5lab/gamelab/Exporter.js [hunk 4]
```diff
@@ -71,55 +53,14 @@ export default {
...
-      });
-
-      zip.file(appName + '/package.json', exportExpoPackageJson);
-      zip.file(appName + '/app.json', appJson);
-      zip.file(appName + '/App.js', appJs);
-      zip.file(appName + '/CustomAsset.js', exportExpoCustomAssetJs);
-      zip.file(appName + '/DataWarning.js', exportExpoDataWarningJs);
-      zip.file(appName + '/metro.config.js', exportExpoMetroConfigJs);
-    }
+    // remove/update comment
```
Comment (epeach) [inline apps/src/p5lab/gamelab/Exporter.js]: Same comment here as the other `remove` comment. Intended to leave? And if so, can we add more context?

File: apps/src/applab/applab.js (hunk 1 above)
Comment (epeach) [reply apps/src/applab/applab.js:389]: Oh, found it here! NVM! https://github.com/code-dot-org/code-dot-org/pull/48224/files#diff-0dc785e6d0108a24cc4e8ca5d7e0359dd8387cf9c620812d438f4ca5f0c656eeL389

File: apps/src/p5lab/gamelab/Exporter.js (hunk 4 above)
Comment (bencodeorg) [reply apps/src/p5lab/gamelab/Exporter.js]: Did not intend to leave, thanks for looking through this in detail! [Found a version of this file from way back](https://github.com/code-dot-org/code-dot-org/blame/5244615f60b6997372ff05f0bf644e621b280677/apps/src/applab/Exporter.js), just before the ["expo needs this to come first" was added](https://github.com/code-dot-org/code-dot-org/blame/06553fabba7ea9d659de1229655afc2d6c2cfd52/apps/src/applab/Exporter.js). Looks like index.html already came first, but for some reason was important to expo to remain that way. Deleted the comment, as well as the one in the Applab exporter.

File: apps/src/p5lab/P5Lab.js (hunk 2 above)
Comment (bencodeorg) [reply apps/src/p5lab/P5Lab.js]: Done, good eye!

File: apps/src/applab/applab.js (hunk 1 above)
Comment (bencodeorg) [reply apps/src/applab/applab.js:389]: Yeah didn't mean to leave this, but I do remove it in the next PR.

**Review by epeach — APPROVED** [review]
:+1:

**Pull request merged**